### PR TITLE
Update app-stores.md

### PR DESCRIPTION
**Pull request by EthanHermsey**
Changing the russian locales to Japanese.

# Why

Obvious reasons.

# How
Translating 'Hello' and 'These words in japanase' to japanese and adding them to the file.

# Checklist
- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/distribution/app-stores.md [hunk 1]
```diff
@@ -113,19 +113,19 @@ If you plan on shipping your app to different countries, regions, or just want i
...
 
 The keys provided to `locales` should be the [2-letter language code](https://www.loc.gov/standards/iso639-2/php/code_list.php) of your desired language, and the value should point to a JSON file that looks something like this:
 
 ```json
-// russian.json
+// japanese.json
 {
-  "CFBundleDisplayName": "Привет",
-  "NSContactsUsageDescription": "Эти слова по русски"
+  "CFBundleDisplayName": "おい",
```
Comment (brentvatne) [inline docs/pages/distribution/app-stores.md:126]:
```suggestion
  "CFBundleDisplayName": "こんにちは",
```

**Review by brentvatne — CHANGES_REQUESTED** [review]
thanks!

File: docs/pages/distribution/app-stores.md [hunk 2]
```diff
@@ -113,19 +113,19 @@ If you plan on shipping your app to different countries, regions, or just want i
...
+  "NSContactsUsageDescription": "日本語のこれらの言葉"
 }
 ```
 
-Now, iOS knows to set the display name of your app to `Привет` whenever it's installed on a device with the language set to Russian.
+Now, iOS knows to set the display name of your app to `おい` whenever it's installed on a device with the language set to Japanese.
```
Comment (wschurman) [inline docs/pages/distribution/app-stores.md:131]:
```suggestion
Now, iOS knows to set the display name of your app to `こんにちは` whenever it's installed on a device with the language set to Japanese.
```
(to match change above)

File: docs/pages/distribution/app-stores.md (hunk 1 above)
Comment (brentvatne) [reply docs/pages/distribution/app-stores.md:126]: suggested by @risako-k

**Pull request merged**